core/src/display_object.rs

```diff
@@ -557,6 +673,97 @@ pub fn render_base<'gc>(this: DisplayObject<'gc>, context: &mut RenderContext<'_
         None
     };
 
+    let mut cache_info: Option<(BitmapHandle, Option<(u32, u32)>)> = None;
+    let base_transform = context.transform_stack.transform();
+    let bounds: Rectangle<Twips> = this.bounds_with_transform(&base_transform.matrix);
```

Comment:
IIRC in some edge cases I recall seeing `bounds_with_transform` on profiles (as it's tree-recursive), so maybe we could avoid calling it here in the common no-cache case?

Reply:
It's already guarded on the later commits